docs/site/tutorials/core/index.md

```diff
@@ -0,0 +1,41 @@
+---
+lang: en
+title: 'Build large scale Node.js projects with LoopBack 4'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/core-tutorial.html
+---
+
+# Build large scale Node.js projects with LoopBack 4
+
+It's super easy to start developing in Node.js, but challenges will get in the
+way as your project grows with more and more modules, components, developers,
+teams, and releases. If you are building on an open framework or large-scale
+application, you probably have increasing number of **hmm** moments in deciding
+between **Be less opinionated** and **Don't repeat yourself**.
```

Comment:
Are these two mutually exclusive?

Reply:
@raymondfeng, since you're the original author of the content, what's your thought? :)

Reply:
@nabdelgadir, While I don't think those two are mutually exclusive, what would be your suggestion?

Reply:
They are not mutually exclusive. It's a balance between covering common tasks and allowing different choices. The separation of `extension point` (common) and `extension` (specific) achieves that.

Reply:
Okay, thanks!